### DISCUSSION TITLE
Meeting of Remix CC (2023-10-23)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#97] [Meeting of Remix CC]
### 议题
暂无

**Comment by drafish** [comment]
### DJQ
1. 接受星火链网1024活动采访 -- https://github.com/PlanckerLabs/remix-cc/discussions/94
2. 与 remix 开会同步Q4工作规划
3. 与remix团队的Filip交流了关于remix vscode的想法 -- https://github.com/PlanckerLabs/remix-cc/discussions/96

**Comment by yiwen4** [comment]
### yiwen
1. 输出一篇 Remix CC 运营计划
   输出一篇  Remix CC  双周报， 公众号栏目改版
   将Q4工作计划 表格化
2. 申请OP赞助
3. 完成9月翻译 review remix 文档
   完成review remix 文档 markdown 解析问题
4. 日常运营：Twitter
